pkgs/top-level/all-packages.nix

```
@@ -7811,6 +7811,8 @@ with pkgs;
 
   onlykey = callPackage ../tools/security/onlykey { node_webkit = nwjs; };
 
+  onlyoffice-documentserver-bin = callPackage ../servers/onlyoffice-documentserver-bin { };
```

```suggestion
  onlyoffice-documentserver = callPackage ../servers/onlyoffice-documentserver { };
```
We are not doing that if there is only one package to choose from.